gui/src/main/java/io/bisq/gui/components/indicator/TxConfidenceIndicator.java

```diff
@@ -67,7 +63,7 @@
  */
 
 @SuppressWarnings({"SameParameterValue", "WeakerAccess"})
-public class TxConfidenceIndicator extends Control {
+public class TxConfidenceIndicator extends ProgressIndicator {
```

Comment:
Could you move the changes for the ProgressIndicator to another separate PR?
There have been some reasons why we used that because the standard ProgressIndicator waste a lot of CPU. I don't remember the details but would require a comparison and test, so I would prefer to defer that for later and keep it separate form the Java 9 PR.